src/form.tsx

```diff
@@ -47,12 +47,12 @@ const Form: React.FC<FormProps> = ({
     const {texts} = useTheme();
     const id = useAriaId(idProp);
 
-    React.useEffect(
-        () => () => {
+    React.useEffect(() => {
+        isMountedRef.current = true;
```

Comment:
please add a short comment explaining why this assignment is necessary

Reply:
done